adam-core/src/main/scala/org/bdgenomics/adam/util/FileExtensions.scala

```diff
@@ -63,7 +63,11 @@ private[adam] object FileExtensions {
    */
   def isFastqExt(pathName: String): Boolean = {
     pathName.endsWith(".fq") ||
-      pathName.endsWith(".fastq")
+      pathName.endsWith(".fastq") ||
+      pathName.endsWith(".fastq.gz") ||
```

Comment:
We removed compressed file extensions from `FileExtensions` in review for commit 93d17b418ddc918ae95095b961ae7ca045be3f5d.  There is an [`ADAMContext.trimExtensionIfCompressed`](https://github.com/bigdatagenomics/adam/blob/master/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala#L1281) method that checks the Hadoop configuration.  Can that still work with these changes?